Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
Lockfile version 3 was introduced in 1.53 which is not stable yet. Can you please remove this line?

Reply:
Fixed!